pages/index.tsx

```diff
@@ -300,17 +300,56 @@ async function getEvents(sheets: sheets_v4.Sheets): Promise<IEventDTO[]> {
   let events: IEventDTO[] = [];
   const rows: string[][] = response.data.values;
   if (rows.length) {
-    events = rows.map((row: string[]) => ({
-      title: row[0],
-      place: row[1] ?? undefined,
-      link: row[2] ?? undefined,
-      start: row[3] + " " + row[4],
-      end: row[5] + " " + row[6],
-      groupId: parseInt(row[7]),
-      filterId: parseInt(row[8]),
-    }));
-
-    return events;
+    const dstStart = new Date(new Date().getFullYear(), 2, 26);
+    const dstEnd = new Date(new Date().getFullYear(), 9, 29);
+
+    events = rows.map((row: string[]) => {
+      let start: Date = new Date(row[3] + " " + row[4]);
+      let end: Date = new Date(row[5] + " " + row[6]);
+
+      start.getTime() > dstStart.getTime() &&
+        start.getTime() < dstEnd.getTime() &&
+        start.setHours(start.getHours() - 1);
+
+      end.getTime() > dstStart.getTime() &&
+        end.getTime() < dstEnd.getTime() &&
+        end.setHours(end.getHours() - 1);
```

Comment:
What is this code doing? These are only checks, not assigned to a variable or something 🤔